ocs_ci/ocs/ui/base_ui.py

```diff
@@ -2210,36 +2268,32 @@ def login_ui(console_url=None, username=None, password=None):
                         login_loc["kubeadmin_login_approval"][1],
                         login_loc["kubeadmin_login_approval"][0],
                     )
-                )
+                ),
+                message="'Log in with kube:admin' text is not present",
             )
         element.click()
-    except TimeoutException as e:
-        take_screenshot()
-        copy_dom()
-        logger.error(e)
-    element = wait.until(
-        ec.element_to_be_clickable((login_loc["username"][1], login_loc["username"][0]))
-    )
-    take_screenshot()
-    copy_dom()
+    except TimeoutException:
+        take_screenshot("login")
+        copy_dom("login")
+        logger.error(traceback.format_stack())
+
+    username_el = wait_for_element_to_be_clickable(login_loc["username"], 60)
     if username is None:
```

Comment:
What does `el` mean in the variable name of username_el, password_el, confirm_login_el & skip_tour_el? If nothing, pls remove it.

Reply:
el stands for element. it is a common naming in ui testing